packages/migrations/src/202111012023800-create-table-deleted-bots.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * This file is part of Serlo.org.
+ *
+ * Copyright (c) 2013-2021 Serlo Education e.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @copyright Copyright (c) 2013-2021 Serlo Education e.V.
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/serlo-org/serlo.org for the canonical source repository
+ */
+import { createMigration } from './utils'
+
+createMigration(exports, {
+  up: async (db) => {
+    await db.runSql(`CREATE TABLE IF NOT EXISTS deleted_bots LIKE user`)
+  },
+  // TODO: @inyono Do I need a down-type?!
+  /* down: async (db) => {
+    await db.runSql(`INSERT INTO type (name) VALUES ('text-hint')`)
+  },*/
```

Comment:
@inyono What is this down-command for? A possible undo for the migration?!

Reply:
thanks, @kulla !